pkg/virt-operator/resource/apply/delete.go

```diff
@@ -464,45 +465,48 @@ func DeleteAll(kv *v1.KubeVirt,
 		}
 	}
 
-	objects = stores.ValidatingAdmissionPolicyBindingCache.List()
-	for _, obj := range objects {
-		if validatingAdmissionPolicyBinding, ok := obj.(*admissionregistrationv1.ValidatingAdmissionPolicyBinding); ok && validatingAdmissionPolicyBinding.DeletionTimestamp == nil {
-			if key, err := controller.KeyFunc(validatingAdmissionPolicyBinding); err == nil {
-				expectations.ValidatingAdmissionPolicyBinding.AddExpectedDeletion(kvkey, key)
-				err := clientset.AdmissionregistrationV1().ValidatingAdmissionPolicyBindings().Delete(context.Background(), validatingAdmissionPolicyBinding.Name, deleteOptions)
-				if err != nil {
-					expectations.ValidatingAdmissionPolicyBinding.DeletionObserved(kvkey, key)
-					log.Log.Errorf("Failed to delete validatingAdmissionPolicyBinding %+v: %v", validatingAdmissionPolicyBinding, err)
-					return err
-				}
-			}
-		} else if !ok {
-			log.Log.Errorf(castFailedFmt, obj)
-			return nil
-		}
+	if err = deleteKubeVirtLabelsFromNodes(clientset); err != nil {
```

Comment:
You are dropping `ValidatingAdmissionPolicyBinding` and `ValidatingAdmissionPolicy` cleanup. Maybe a rebase mistake